src/app/(dashboard)/(routes)/subject-dashboard/page.tsx

```diff
@@ -76,7 +77,7 @@ const PageContent = () => {
       eventAction: "subject_opened",
       label: userId ? "student" : "guest",
       label1: userId || null,
-      label2: quizPath,
+      label2: subject=="mathematics" ? "math" :subject?.toLocaleLowerCase()||"",
```

Comment:
Why is this changed?

Reply:
this quizpath have values of all the subjects not the newly added  languages , that's why I changed this